frontend/src/App.jsx

```diff
@@ -15,12 +17,15 @@ function App() {
           <Link to="/">Home</Link>
           <Link to="/profile">Profil</Link>
           <Link to="/contact">Contact</Link>
+          <Link to="/FourGlass">FourGlass</Link>
         </nav>
       </header>
 
       <main className="App">
         <Routes>
           <Route path="/" element={<Home />} />
+          <Route path="/Questionnaire" element={<Questionnaire />} />
+          <Route path="/FourGlass" element={<FourGlass />} />
```

Comment:
urls should be in kebab-case, not PascalCase

Reply:
change made ;-)